Cargo.toml

```diff
@@ -57,4 +59,11 @@ syn = { version = "1.0.105", features=["full", "extra-traits", "visit"] }
 base64 = "0.13.1"
 bigdecimal = "0.3.0"
 num-bigint = "0.4.3"
-serde_bytes = "0.11.5"
\ No newline at end of file
+serde_bytes = "0.11.5"
+
+[patch.crates-io]
+polywrap_core = { path = "packages/core" }
```

Comment:
:0 this fixes things?

Reply:
Looks like it. Everything builds, tests pass and I've also tested it in a separate branch with the plugin.into() issue I was having and it was no longer there